sim/hunter/marksmanship/TestMM.results

```diff
@@ -2158,8 +2179,8 @@ dps_results: {
 dps_results: {
  key: "TestMM-AllItems-WyrmstalkerBattlegear"
  value: {
-  dps: 24191.33598
-  tps: 21766.91344
+  dps: 24506.21055
+  tps: 22081.788
  }
```

Comment:
This doesn't look RNG seed related, maybe worth a closer look?

Reply:
Yep, copy pasta mistake.